src/types/linked_accounts.ts

```diff
@@ -48,6 +48,7 @@ export type LinkedAccount = {
   connection_external_id?: string
   connection_needs_repair_as_of: string | null
   is_syncing: boolean
+  opening_account_balance_missing?: boolean
```

Comment:
Change later this field to know when "Add opening balance" option should be shown